<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TOptionGeneral</class>
 <widget class="QWidget" name="TOptionGeneral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>128</height>
      </size>
     </property>
     <property name="title">
      <string>Gui</string>
     </property>
     <widget class="QCheckBox" name="ckbAutoPlay">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>54</y>
        <width>246</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Auto play last track after started</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="ckbTrayIcon">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>77</y>
        <width>168</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Display system tray icon</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="ckbMultiInstance">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>168</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Allow multiple instances</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>52</width>
        <height>16</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>52</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Language:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cmbLanguage">
      <property name="geometry">
       <rect>
        <x>79</x>
        <y>30</y>
        <width>271</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>114</height>
      </size>
     </property>
     <property name="title">
      <string>Duration correct</string>
     </property>
     <widget class="QRadioButton" name="rbAuto">
      <property name="geometry">
       <rect>
        <x>115</x>
        <y>31</y>
        <width>47</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Auto</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
     <widget class="QRadioButton" name="rbForce">
      <property name="geometry">
       <rect>
        <x>189</x>
        <y>31</y>
        <width>53</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Force</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
     <widget class="QRadioButton" name="rbDisable">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>31</y>
        <width>65</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Disable</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
     <widget class="QSpinBox" name="sbCheckDuration">
      <property name="geometry">
       <rect>
        <x>119</x>
        <y>54</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="suffix">
       <string>ms</string>
      </property>
      <property name="minimum">
       <number>1000</number>
      </property>
      <property name="maximum">
       <number>60000</number>
      </property>
      <property name="singleStep">
       <number>1000</number>
      </property>
      <property name="value">
       <number>3000</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>54</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Check duration:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QTimeEdit" name="tePlayDuration">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="displayFormat">
       <string notr="true">HH:mm:ss</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>80</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Play duration:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="title">
      <string>Export</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>26</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Parallel processes:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbParallels">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>26</y>
        <width>44</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>4</number>
      </property>
     </widget>
     <widget class="QCheckBox" name="ckbAutoClearMission">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>361</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Auto clear completed missions.</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
